### PR TITLE
Fixes #16495 - foreman_openscap 0.6 manual

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: plugins/foreman_openscap/0.6/index.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
+---
+layout: plugin
+title: Foreman OpenSCAP manual
+images: /plugins/foreman_openscap/0.6
+version: 0.4.1
```
Comment (shlomizadok) [inline plugins/foreman_openscap/0.6/index.md]: Should this be 0.6 ?

Comment (ares) [reply plugins/foreman_openscap/0.6/index.md]: fixed also in 0.5 directory

**Comment by ares** [comment]
could someone take a look please? :-) 0.6.2 is out for some time

**Comment by shlomizadok** [comment]
LGTM :+1:

**Comment by mmoll** [comment]
merged, díky @ares!

**Pull request merged**